Source/Core/InputCommon/DynamicInputTextureConfiguration.cpp

```diff
@@ -300,24 +300,42 @@ bool DynamicInputTextureConfiguration::GenerateTexture(
     }
   }
 
-  // Load image copy
-  auto base_image = LoadImage(m_base_path + texture_data.m_image_name);
-  bool dirty = false;
+  // Load source texture twice
+  // once as an original image to fallback to if a key or device isn't mapped
+  // the second is the image to write to the textures directory
+  auto original_image = LoadImage(m_base_path + texture_data.m_image_name);
+  auto image_to_write = LoadImage(m_base_path + texture_data.m_image_name);
 
+  bool dirty = false;
   for (auto& [emulated_key, rects] : emulated_controls_iter->second)
   {
-    std::string host_key = "";
-    sec->Get(emulated_key, &host_key);
+    auto apply_original = [&, rects = std::ref(rects)] {
```

Comment:
In case anyone is curious why this is needed, see:  https://stackoverflow.com/questions/46114214/lambda-implicit-capture-fails-with-variable-declared-from-structured-binding

Reply:
I wonder if this is good enough of a reason to leave a comment in source 🤔 
At least I didn't know this was a thing (nor have i seen `std::ref` before); but I've never ran into the issue either.

Alternatively...declaring a local ref inside the `for` loop to get around this? At least that sounds like a way to fix this with non-obscure syntax.

Reply:
Yeah, maybe consider adding a `// TODO: Remove the initializer once we can use C++20` comment like we did in the GC memcard code when we hit this issue last time.